src/modules/party-external-rating/party-external-rating.module.ts

```diff
@@ -0,0 +1,21 @@
+import { DynamicModule, Module } from '@nestjs/common';
+
+import { PartyExternalRatingController } from './party-external-rating.controller';
+import { PARTY_EXTERNAL_RATINGS_PROVIDER_SYMBOL, PartyExternalRatingModuleOptions } from './party-external-rating.module-definition';
+
+@Module({})
+export class PartyExternalRatingModule {
```

Comment:
I'm curious if we need it to be a dynamic Model? or it can be static?

Reply:
@lmarrai-sw  I had a chat with @abhi-markan  on this. looking at the roadmap for ACBS, we can safely use static modules for TFS. We are using a similar approach for MDM

Reply:
@IgnatG I've changed this to be a static module.

The structure I've done for is 
- functionality for interacting with ACBS (i.e. making the GET `/Party/{partyId}/PartyExternalRating` request) lives in the `acbs` module
- the `party-external-rating` module imports and depends on the `acbs` module directly, it transforms the response from ACBS for use in our api

There's also a small `acbs-adapter` module for anything global we need to do to make `acbs` suitable for use with our API (currently it just holds the `AcbsExceptionTransformInterceptor` that is for transforming exceptions from the `acbs` module into HTTP exceptions). Only the `tfs` module needs to import/depend on it

Reply:
Let me know if you're happy with that :) 

I'd also be interested to understand why you and Abhi preferred this approach to the dynamic module approach - it's not urgent to discuss though and we can discuss over Slack if you prefer